interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -362,6 +371,10 @@ protected ManagedChannel createChannel() {
 
   protected abstract ManagedChannelBuilder<?> createChannelBuilder();
 
+  protected LoadBalancerProvider getOrcaLoadBalancerProvider() {
```

Comment:
What is the point of this method, if it does nothing here and the only caller that implements it could just call a method of its own?